sdk/nextjs/src/pages/bucketing.ts

```diff
@@ -49,6 +48,8 @@ export const getBucketedConfig = async (
         sdkKey,
         'bootstrap',
         obfuscated,
+        '',
```

Comment:
so we're no longer allowing the `lastModifiedThreshold` to be passed in?

Reply:
it never was